clients/js/src/generated/instructions/burn.ts

```diff
@@ -87,7 +87,7 @@ export function getBurnInstructionDataSerializer(): Serializer<
 }
 
 // Extra Args.
-export type BurnInstructionExtraArgs = { proof: Array<PublicKey> };
+export type BurnInstructionExtraArgs = { proof?: Array<PublicKey> };
```

Comment:
I'm noticing a lot of these cases where what were required args are becoming optional. I know these are fields injected in the Kinobi config, is there some default-to-optional setting that's happening or is this intended behavior?